app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    NotWearingMaskError,
+    OutdatedVaccineError,
+    NotVaccinatedError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    count_friends = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except (OutdatedVaccineError, NotVaccinatedError):
+            return "All friends should be vaccinated"
```

Comment:
Returning 'All friends should be vaccinated' inside the loop will cause the function to exit on the first friend who is not vaccinated or has an outdated vaccine. This means not all friends are checked. Consider using a flag or collecting results to ensure all friends are processed before returning a message.